indico/modules/events/management/blueprint.py

```diff
@@ -40,6 +41,8 @@
 _bp.add_url_rule('/event-location', 'manage_event_location', RHManageEventLocation, methods=('GET', 'POST'))
 _bp.add_url_rule('/event-keywords', 'manage_event_keywords', RHManageEventKeywords, methods=('GET', 'POST'))
 _bp.add_url_rule('/event-persons', 'manage_event_person_links', RHManageEventPersonLinks, methods=('GET', 'POST'))
+_bp.add_url_rule('/print-poster/settings', 'poster_settings', RHPosterPrintSettings, methods=('GET', 'POST'))
```

Comment:
Because it was initially optional. You could also open the dialog without a pre-selected template. And if you think of it, it still kind of makes sense URL-wise.